src/openms/source/ANALYSIS/OPENSWATH/TransitionTSVFile.cpp

```diff
@@ -258,6 +263,26 @@ namespace OpenMS
                                          " do not match: " + String(tmp_line.size()) + " != " + String(header_dict.size()));
       }
 
+
+      // Check for ambiguous entity type: more than one of ProteinId, OligoName, CompoundName not empty
```

Comment:
@jcharkow It looks like we don't have any logic to prevent transitions being more than one type. Is it reasonable to test for that here?